### PR TITLE
Adding Matteo Ferrea's Cincinnati Chili I March Recipe

### PR DESCRIPTION
**Pull request by FerrMattDev**
Merging March recipes update

### CODE REVIEW

**Comment by impute** [comment]
1) Verified merge from their branch to Emile's develop
> mattemalmsteen wants to merge 1 commit into emileastih1:develop from mattemalmsteen:cincinnati-chili-mar

2) Nothing important, just informational of merge.

3) Double spaced ingredients but no problem; looks fine.

File: app/recipe/mar/.Cincinnati_Chili.md.swp
Comment (impute) [inline app/recipe/mar/.Cincinnati_Chili.md.swp:1]: What is this?

**Review by FerrMattDev — COMMENTED** [review]
Some issues to check, see details

File: app/recipe/mar/.Cincinnati_Chili.md.swp
Comment (FerrMattDev) [reply app/recipe/mar/.Cincinnati_Chili.md.swp:1]: I deleted a previous files, that seems left some trace behind.

**Comment by FerrMattDev** [comment]
All done

**Pull request merged**